### ISSUE TITLE
Labelling: Show list of current labellings + make them sortable

### ISSUE THREAD

**Issue by bfrgottscma** [SiLeBAT/fcl-client#558]
Please display the current labellings from the dataset in the labelling list.

As a user I would like to be able to sort the labelling conditions manually (only one at the same time). Make the list sortable via drag and drop (as in color&shape highlighting).
- Moving should only be possible inside the labelling condition list if possible --> only in list should be drop target; in other positions x to show you cant drop it here
- The order in the condition list AND in the legend should be affected by sorting. Tooltip (move up/down).
- Sorting of legend: Stations have priority if you have the "same" condition for stations and deliveries (e.g. forward trace).

**Think of Tests**
- [ ] Excel testsheet needed
- [ ] E2e test needed
- [ ] Integration test needed
- [ ] Unit test needed

**Additional context**
- [ ] Mockup needed

**Comment by afalenski** [comment]
- Labels are sortable as described above.
- Labels are not shown in the legend. @bfrgottscma Do you really want to see labelling rules in the legend? I do not think this would add value to the WebApp.
For me this ticket is done. I would delete bullet point 3 and the part about the legend in bullet point 2 in the ticket text.
______
**Tested with**
OS: Windows 10

Datasets: 
- Example Dataset

Browsers:
- Google Chrome Version 92.0.4515.159 (Offizieller Build) (64-Bit)
- Firefox Quantum 68.9.0esr (64-Bit)

**Comment by bfrgottscma** [comment]
Labels sortable, sorting affects order of labels in graph, different labels are separated by /
All points concerning the legend are outdated. We decided not to show labels in the legend after the ticket was created (sorry, my bad for not adapting the ticket).
We do not need the tooltips (neither in color/shape section) since the mouse becomes an up-/downfacing arrow when hovering over the six dots. (Specification to add tooltips is outdated, sorry my bad as well)

Done

Tested with Example Data
Windows 10
Firefox (91.0.1 (64-Bit)) and Chrome (Version 92.0.4515.159 (Offizieller Build) (64-Bit))
